New envelope zooming actions

wolqws opened the pull request:
SWS/wol: Full zoom selected envelope (in media lane only) to upper/lower half in time selection
SWS/wol: Vertical zoom selected envelope (in media lane only) to upper/lower half
SWS/wol: Put selected envelope in media/envelope lane

Jeff0S commented:
Thank you for the contribution wolqws!

Weird... This pull request was made on the "master" branch although it included things from my next branch... Anyway, I have cherry-pick'd your relevant commit, so it should be ok.
For your next contributions, please make you pull request on the next branch (so we can just use the "Merge pull request" button on github). TIA!

wolqws commented:
Oh yeah, have not seen it was on master, ofc should have been to next. It included your next branch because I have updated my next branch from repo, added my commits and opened a pull request to master. If I've requested to merge to next only my commits would have been here.
